apps/src/openni_klt.cpp

```diff
@@ -295,7 +295,7 @@ class OpenNIViewer {
   typename pcl::tracking::PyramidalKLTTracker<PointType>::Ptr tracker_;
   pcl::PointCloud<pcl::PointUV>::ConstPtr keypoints_;
   pcl::PointIndicesConstPtr points_;
-  pcl::PointIndicesConstPtr points_status_;
+  pcl::shared_ptr<const std::vector<int>> points_status_;
```

Comment:
This seems counter intuituve

Reply:
As described in the commit message, it has nothing to do with indices. The status can be 0, -1, or -2, describing whether the point was successfully tracked or not (and why not).